### PR TITLE
Migration docs updates

### PR DESCRIPTION
**Pull request by ysuarez**
## Purpose / why

I wanted the Islandora 8 docs site to have links to more recent migration related videos that the community / foundation has recently created.

Related to issue #1579 

## What changes were made?

I added links to a newer Youtube video that covers migrations in "migrate-7x.md". 

Islandora Webinar: Migrating from Islandora 7 to Islandora 8
https://www.youtube.com/watch?v=n0kLOB8ZdbM
Nov 21, 2019

Also a new page "migration-overview.md" and added content that summarizes another migration related video.  

Islandora Online: Islandora Migration Tools
https://www.youtube.com/watch?v=95Bnix-z1zY&feature=em-uploademail
Aug 10, 2020

## Verification

Please suggest changes to overall content, spelling, or grammar.

## Interested Parties

@manez @dannylamb 

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [x] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [x] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [x] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

File: docs/technical-documentation/migration-overview.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+- Updating metadata is as simple as:
+`drush mim node --update`
+
+A Migration Configuration defines an Extract, Transform, Load (ETL) process.
+
+- Source plugins extract data from a source
+- Process plugins transform the data
+- Destination plugins load the data (create new entities)
+
+We’ve built two tools for you using the Migrate API:
```
Comment (manez) [inline docs/technical-documentation/migration-overview.md]: I found this section a bit hard to follow (there's a lot of information, which is awesome!). Can you add some sub-headings to break it out visually? I think if "A Migration Configuration defines an Extract, Transform, Load (ETL) process." was ####, it would read easier.

File: docs/technical-documentation/migration-overview.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+- [migrate_islandora_csv](https://github.com/Islandora/migrate_islandora_csv)
+  - Tutorial with a sample migration using some files and a CSV
+- [migrate_7x_claw](https://github.com/Islandora-Devops/migrate_7x_claw)
+  - A tool to get all your Islandora 7 content migrated over
+
+Recap of migrate_islandora_csv:
```
Comment (manez) [inline docs/technical-documentation/migration-overview.md]: I did some testing, and headlines can go as deep as 6 #'s, so these sub-section could be headlined too.

File: docs/technical-documentation/migration-overview.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- CSVs
+  - Everyone understands and knows how to work with CSVs
+- Documented
+  - It’s a step by step walkthrough
```
Comment (manez) [inline docs/technical-documentation/migration-overview.md]: "step-by-step"?

**Review by manez — CHANGES_REQUESTED** [review]
@ysuarez The new page is great! The summary for each section of the video is very, very helpful. I have a couple of minor suggestions, but overall, really nice addition. Thank you!

File: docs/technical-documentation/migration-overview.md (hunk 3 above)
Comment (ysuarez) [reply docs/technical-documentation/migration-overview.md]: Good catch

File: docs/technical-documentation/migration-overview.md (hunk 1 above)
Comment (ysuarez) [reply docs/technical-documentation/migration-overview.md]: I added more Markdown headings marks to the docs. Let me know what you think.

**Review by manez — APPROVED** [review]

**Comment by manez** [comment]
Looks good @ysuarez !

My computer's in the shop and I'm using a temp machine that I can't build docs on, so I'm going to hold off on merging — unless some one else in @Islandora/8-x-committers can spin this up and make sure it all builds in mkdocs?

File: docs/technical-documentation/migration-overview.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+- /media/{mid}/source
+  - PUT a file to this endpoint to create/update a Media’s file
+- /node/{nid}/media/{media_type}/{taxonomy_term}
+  - PUT a file to this endpoint to create/update a Media for a Node
+
+Just be aware, you are writing everything yourself! (In other words you are making all of the migration decisions yourself.)
+
+### Migrate API
+
+Uses the Drupal 8 [Migrate API](https://www.drupal.org/docs/8/api/migrate-api/migrate-api-overview), which "provides services for migrating data from a source system to Drupal 8.
```
Comment (whikloj) [inline docs/technical-documentation/migration-overview.md:35]: This builds fine and displays nicely. Just wondering about the start of this sentence. It feels like it is missing something?

File: docs/technical-documentation/migration-overview.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+- Command Line Skills
+  - This is best done with shell access and drush
+
+#### Migrate API demo video
+
+Check out this video that demonstrates the Drupal Migrate API migration process: [Islandora Webinar: Migrating from Islandora 7 to Islandora 8](migrate-7x.md)
+
+### Islandora Workbench
+
+https://github.com/mjordan/islandora_workbench
```
Comment (whikloj) [inline docs/technical-documentation/migration-overview.md:101]: Could you make this URL a hyperlink to its location. Just so people can click through

File: docs/technical-documentation/migrate-7x.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
-## Migrating from Islandora 7.x to Islandora CLAW (YouTube Video)
+## Migrating from Islandora 7.x to Islandora 8 (YouTube Video)
 
-[![Migrating from Islandora 7.x to Islandora CLAW](https://img.youtube.com/vi/fEDzfSPjKEo/0.jpg)](https://www.youtube.com/watch?v=fEDzfSPjKEo)
+[![Islandora Webinar: Migrating from Islandora 7 to Islandora 8](https://img.youtube.com/vi/n0kLOB8ZdbM/0.jpg)](https://www.youtube.com/watch?v=n0kLOB8ZdbM)
+Nov 21, 2019
 
 For tools to migrate into Islandora 8 from an existing Islandora 7 instance, please see [migrate_7x_claw](https://github.com/Islandora-devops/migrate_7x_claw)
```
Comment (whikloj) [inline docs/technical-documentation/migrate-7x.md:6]: While you're in here, seems like this should point to your new Migration Overview page instead or at least reference the CSV migration module.

**Comment by rosiel** [comment]
LGTM. Thank you @ysuarez!

File: docs/technical-documentation/migration-overview.md (hunk 4 above)
Comment (rosiel) [reply docs/technical-documentation/migration-overview.md:35]: It reads consistent with the rest of the document. Improvements can always happen later.

**Comment by whikloj** [comment]
You all can go ahead and merge it then, I don't want to hold anything up.

**Comment by rosiel** [comment]
Oops - failed to deploy - invalid option "--token=". 

https://travis-ci.com/github/Islandora/documentation/jobs/381154974

I'll see if i can kick at this and get it up on http://islandora.github.io/documentation/ !

**Comment by rosiel** [comment]
Travis needed a github token that was set to the 'main' branch, not 'master'. 

Check out this beautiful work! http://islandora.github.io/documentation/technical-documentation/migration-overview/
